crates/primitives/src/constants.rs

```diff
@@ -29,6 +29,6 @@ pub const TEST_PRIVATE_KEY: &str =
     "1212121212121212121212121212121212121212121212121212121212121212";
 
 pub const MAX_STATEDIFF_SIZE_COMMITMENT_THRESHOLD: u64 = 300 * 1024;
-pub const MAX_STATEDIFF_SIZE_PROOF_THRESHOLD: u64 = 400 * 1024;
+pub const MAX_STATEDIFF_SIZE_PROOF_THRESHOLD: u64 = 397000;
```

Comment:
Can we use the constant from bitcoin-da instead?

Reply:
Sure, I'll make it pub and use it

Reply:
can we put this to citrea-primitives so that both the prover and the bitcoin-da can pull it from there?

citrea prover importing something from bitcoin-da does not really make sense